_includes/docs/pe/user-guide/integrations/remote-integrations.md

```diff
@@ -8,46 +8,47 @@
 * TOC
 {:toc}
 
-## Introduction
+It is possible to execute any ThingsBoard Integration remotely from main ThingsBoard instance. This guide contains step-by-step instructions how to launch ThingsBoard integration remotely.
```

Comment:
Please add more information about why remote integration was invented. This is for scenarios when one needs to connect to some local service that is behind NAT.